### PR TITLE
Navigate branches

### PR DESCRIPTION
**Pull request by sindrig**
I'm constantly trying to use , and . to navigate between branches in the branch tab. This is basically just a copy of the implementation used in the status tab.

### CODE REVIEW

File: syntax/status.sublime-syntax [hunk 1]
```diff
@@ -39,6 +39,17 @@ contexts:
...
+      comment: branch-names
+      captures:
+        0: keyword.other.git-savvy.branch-header
+      push:
+        - meta_scope: meta.git-savvy.branch.section
+        - match: ^$
+          pop: true
+        - match: '^  . [A-Za-z0-9]{7} (.+)\n$'
+          captures:
+              1: meta.git-savvy.branches.branch gitsavvy.gotosymbol
```
Comment (stoivo) [inline syntax/status.sublime-syntax]: I don't understand why you updated `syntax/status.sublime-syntax`. Would you explain why you did it? Or show one example of what is fixes?

**Comment by sindrig** [comment]
@stoivo This is the first time I'm working on *.sublime-syntax, but I think I'm creating a selector `meta.git-savvy.branches.branch` to match in `branch.py:743`, the same as the one above (`meta.git-savvy.status.file`). I could be mistaken though?

**Comment by stoivo** [comment]
Yes, you are creating a selector, but to the status view. You are editing **status**.sublime-syntax.

**Comment by sindrig** [comment]
Oh shit, sorry about that! Didn't notice that every tab had it's own syntax. I've removed these changes!

**Comment by sindrig** [comment]
So it was only working because of a coincidence, I chose the same selector name as in branch.sublime-syntax!

**Comment by stoivo** [comment]
Do you know why you have these two commits in you PR?
 Merge pull request #1 from divmain/master … bd5fddb
 Merge branch 'master' of github.com:divmain/GitSavvy	 be0bd1b

Maybe you can squash them to remove them?

File: Default.sublime-keymap [hunk 2]
```diff
@@ -613,6 +613,22 @@
...
+        ]
+    },
+    {
+        "keys": [","],
+        "command": "gs_branches_navigate_branches",
+        "args": { "forward": false },
+        "context": [
+            { "key": "setting.git_savvy.branch_view", "operator": "equal", "operand": true }
+        ]
+    },
```
Comment (stoivo) [inline Default.sublime-keymap]: We are making GitSavvy vintageous friendly you also need to change this to 
```js
    {
        "keys": ["."],
        "command": "gs_branches_navigate_branches",
        "args": { "forward": true },
        "context": [
            { "key": "setting.command_mode", "operator": "equal", "operand": false },
            { "key": "setting.git_savvy.branch_view", "operator": "equal", "operand": true }
        ]
    },
    {
        "keys": [","],
        "command": "gs_branches_navigate_branches",
        "args": { "forward": false },
        "context": [
            { "key": "setting.command_mode", "operator": "equal", "operand": false },
            { "key": "setting.git_savvy.branch_view", "operator": "equal", "operand": true }
        ]
    },
    {
        "keys": ["j"],
        "command": "gs_branches_navigate_branches",
        "args": { "forward": true },
        "context": [
            { "key": "setting.command_mode", "operator": "equal", "operand": false },
            { "key": "setting.git_savvy.vintageous_friendly", "operator": "equal", "operand": true },
            { "key": "setting.git_savvy.branch_view", "operator": "equal", "operand": true }
        ]
    },
    {
        "keys": ["k"],
        "command": "gs_branches_navigate_branches",
        "args": { "forward": false },
        "context": [
            { "key": "setting.command_mode", "operator": "equal", "operand": false },
            { "key": "setting.git_savvy.vintageous_friendly", "operator": "equal", "operand": true },
            { "key": "setting.git_savvy.branch_view", "operator": "equal", "operand": true }
        ]
    },
```

**Comment by divmain** [comment]
This PR looks great, btw.  Add the final context changes to the keymap file, and we'll merge this in.

**Comment by sindrig** [comment]
Closing this PR and I've created a new one: https://github.com/divmain/GitSavvy/pull/401

**Pull request closed without merging**